### PR TITLE
Gql01 user create apitests s.swift

### PR DESCRIPTION
**Pull request by dalaran78**
userCreate API tests

### CODE REVIEW

**Comment by dalaran78** [comment]
Closing this PR as pushed spec file only, without supporting files.  Will open new PR

File: src/modules/user/_test/userCreate.spec.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    lastName
+  }
+}`,
+        variables: arg
+      }
+
+      request(graphQLEndpoint)
+        .post('/')
+        .send(postData)
+        .expect(200)
```
Comment (AzamatMurzabay) [inline src/modules/user/_test/userCreate.spec.js]: please change to code 400

File: src/modules/user/_test/userCreateSwi.spec.js [hunk 2]
```diff
@@ -58,7 +56,6 @@ describe("USER CREATE", () => {
         .end((err, res) => {
```
Comment (AzamatMurzabay) [inline src/modules/user/_test/userCreateSwi.spec.js:56]: Error: expected 200 "OK", got 400 "Bad Request"

File: src/modules/user/_test/userCreateSwi.spec.js [hunk 3]
```diff
@@ -75,7 +72,6 @@ describe("USER CREATE", () => {
...
```
Comment (AzamatMurzabay) [inline src/modules/user/_test/userCreateSwi.spec.js:72]: Error: expected 200 "OK", got 400 "Bad Request"

File: src/modules/user/_test/userCreateSwi.spec.js [hunk 4]
```diff
@@ -58,7 +56,6 @@ describe("USER CREATE", () => {
...
           if (err) return done(err);
           const respData = res.body.data;
```
Comment (AzamatMurzabay) [inline src/modules/user/_test/userCreateSwi.spec.js:58]: The expect have to be an error message as example below: 

const respData = res.body.errors[0]
                    console.log("====", respData)
                    expect(respData.message).eq('GraphQL operations must contain a non-empty `query` or a `persistedQuery` extension.')
                    expect(respData.extensions.code).to.eq('INTERNAL_SERVER_ERROR')

File: src/modules/user/_test/userCreateSwi.spec.js [hunk 5]
```diff
@@ -75,7 +72,6 @@ describe("USER CREATE", () => {
...
```
Comment (AzamatMurzabay) [inline src/modules/user/_test/userCreateSwi.spec.js:74]: The expect have to be an error message as example below: 

const respData = res.body.errors[0]
                    console.log("====", respData)
                    expect(respData.message).eq('GraphQL operations must contain a non-empty `query` or a `persistedQuery` extension.')
                    expect(respData.extensions.code).to.eq('INTERNAL_SERVER_ERROR')

**Review by AzamatMurzabay — CHANGES_REQUESTED** [review]

**Pull request closed without merging**